### PR TITLE
add observability docs

### PR DESCRIPTION
**Pull request by kenakamu**
In this PR, I added observability related docs.

I may add more information as sub topic later but please start reviewing.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ The posts in this folder are intended for sharing technical learnings from some
 ```bash
 ├─ architecture # Azure ML architecture information 
 ├─ media # contains images for docs
+├─ observability # contains Azure resouces observability information
```
Comment (liupeirong) [inline docs/README.md:15]: Should we put observability inside "architecture", together with the other architecture related docs in the root folder?

File: docs/observability/MonitorAzureResources.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- [Application logs](https://docs.microsoft.com/en-us/azure/azure-monitor/insights/insights-overview) and [system metrics](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/data-platform#metrics) from all services are exported to [Azure Monitor](https://docs.microsoft.com/en-us/azure/azure-monitor/overview)
+- Native Azure components export directly to Azure Monitor
+
+## Data Scientist
+
+Data Scientists uses Azure Machine Learning to build and publish models. They should have full access to Azure Machine Learning workspace with detail logs and charts. In addition, they will monitor Model Training and Evaluation results using Azure Dashboards, with alert.
+
+- [Get context in Run environment](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-with-datasets#directly-access-datasets-in-your-script) and [add ML train metrics to Application Insight telemetry](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-debug-pipelines#logging-options-example)
```
Comment (liupeirong) [inline docs/observability/MonitorAzureResources.md:34]: The first link doesn't seem to be related to Run Context?

File: docs/observability/MonitorAzureResources.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- [Training Pipeline Metrics](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-log-pipelines-application-insights) are exported Application Insights log or OpenCensus format
+
+## Quality Assurance
+
+Quality Assurance team may run the model performance test build by Data Scientist team by using different set of test dataset. You may want to restrice their access to Azure Machine Learning service so that they cannot see the code to build the model, as they just need to run the model accuracy tests.
```
Comment (liupeirong) [inline docs/observability/MonitorAzureResources.md]: "buil**d** by the Data Scientist..." -> "buil**t** by the Data Scientist..."
"restrice" -> "restict"

File: docs/README.md (hunk 1 above)
Comment (kenakamu) [reply docs/README.md:15]: we separated observability docs from architecture in our project as its big enough topic. But if you prefer move it to architecture, i am fine with it.

File: docs/observability/MonitorAzureResources.md (hunk 2 above)
Comment (kenakamu) [reply docs/observability/MonitorAzureResources.md:34]: actually it is :) it contains python script which uses run context

File: docs/observability/MonitorAzureResources.md (hunk 3 above)
Comment (kenakamu) [reply docs/observability/MonitorAzureResources.md]: fixed .

**Review by liupeirong — APPROVED** [review]
